### PR TITLE
Fix video poster default frame to be the middle frame

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: docs/iframe.html [hunk 1]
```diff
@@ -57,4 +57,4 @@
     console.warn('unable to connect to parent frame for connecting dev tools');
   }</script></head><body><div class="sb-nopreview sb-wrapper"><div class="sb-nopreview_main"><h1 class="sb-nopreview_heading sb-heading">No Preview</h1><p>Sorry, but you either have no stories or none are selected somehow.</p><ul><li>Please check the storybook config.</li><li>Try reloading the page.</li></ul></div></div><div class="sb-errordisplay sb-wrapper"><div id="error-message" class="sb-heading"></div><pre class="sb-errordisplay_code">
```
Comment (asisayag2) [inline docs/iframe.html]: @maoznir , are those supposed to be committed?

File: src/components/Video/Video.js [hunk 2]
```diff
@@ -17,52 +12,71 @@ class Video extends CloudinaryComponent {
...
+
+  getSourceUrlOptions = (childTransformations, sourceTransformations, sourceType) => {
+    const sourceTransformation = sourceTransformations[sourceType] || {};
+    return Util.defaults({}, sourceTransformation, childTransformations, {
+      resource_type: 'video',
+      format: sourceType
+    });
+  };
+
+  getVideoTagProps = () => {
```
Comment (asisayag2) [inline src/components/Video/Video.js]: @maoznir , please add comments so one can understand what's going on

File: src/components/Video/Video.js [hunk 3]
```diff
@@ -17,52 +12,71 @@ class Video extends CloudinaryComponent {
...
-        {this.props.children}
+        {children}
       </video>
     );
   }
 }
+
 Video.propTypes = {publicId: PropTypes.string};
 Video.defaultProps = {};
-Video.contextTypes = CloudinaryComponent.contextTypes;
```
Comment (asisayag2) [inline src/components/Video/Video.js]: @maoznir , so... no 'poster' handling at all? it's all handled by core? 
Are all poster options covered by tests?

File: src/components/Video/Video.js (hunk 2 above)
Comment (maoznir) [reply src/components/Video/Video.js]: done

File: src/components/Video/Video.js (hunk 3 above)
Comment (maoznir) [reply src/components/Video/Video.js]: all attributes except for source handled by core + added tests.
This way we get an alignment with angular sdk, except in react sdk's video component we don't manipulate the dom directly

**Pull request merged**